main.js

```diff
@@ -0,0 +1,79 @@
+//  ==CLOSE BUTTON FOR MENU===//
+const hamburger = document.getElementById('hamburger');
```

Comment:
- [ Optional] I really like the nice mobile nav you created. However, could you make it cover the whole screen and stop the body from scrolling?
![image](https://user-images.githubusercontent.com/44057863/169669149-c25a2855-513e-44f6-af08-a573c5b780e0.png)